README.md

```diff
@@ -60,6 +60,19 @@ WebNN Polyfill requires setting backend to enable TensorFlow.js.
     await tf.ready();
 ```
 
+* When running in browsers, using WASM backend with threads.
```

Comment:
How about:

* When running in browsers with WASM backend, it will use the number of logical CPU cores as the
threads count by default. You can use the `setThreadsCount` API to manually set it.